mod/wikirate_source/spec/set/type/source/source_spec.rb

```diff
@@ -61,182 +85,184 @@
 
       end
     end
-    context "while creating without anything" do
+    context 'while creating without anything' do
       it do
-        sourcepage = Card.new :type_id=>Card::SourceID
+        sourcepage = Card.new type_id: Card::SourceID
         expect(sourcepage).not_to be_valid
         expect(sourcepage.errors).to have_key :source
-        expect(sourcepage.errors[:source]).to include("Please at least add one type of source")
+        expect(sourcepage.errors[:source]).to include('Please at least add one type of source')
       end
     end
-    context "while creating with more than one source type " do
+    context 'while creating with more than one source type ' do
       it do
         url = 'http://www.google.com/?q=wikirateissocoolandawesomeyouknow'
 
-        sourcepage = Card.new :type_id=>Card::SourceID,:subcards=>{ '+Link' => {:content=> url},'+File' =>{:type_id=>Card::FileID}, '+Text' => {:content=>"Hello boys!",:type_id=>Card::BasicID} }
+        sourcepage = Card.new source_args(link: url, text: "Hello boys!")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.